tools/src/test/python/dlpx/virtualization/_internal/conftest.py

```diff
@@ -408,6 +409,7 @@ def mount_specification(virtual_source, repository):
     virtual.status_impl = None
     virtual.initialize_impl = None
     virtual.cleanup_impl = None
+    virtual.source_size_impl = None
```

Comment:
This CP or may be a follow up - do you think it makes sense to add some unit tests in `platform/src/test/python/dlpx/virtualization/test_plugin.py` ?

Reply:
Sure, will be taking care in separate JIRA.